dataprofiler/tests/labelers/test_column_name_model.py

```diff
@@ -0,0 +1,292 @@
+                    }
+                ],
+                # fails, requires `true_positive_dict`
+            },
+            {
+                "false_positive_dict": {}, # fails, required type list
+                "true_positive_dict": [
+                    {
+                        "attribute": "test_attribute",
+                        "label": "test_label"
+                    }
+                ]
+            },
+        ]
+
+        model = ColumnNameModel(parameters=parameters)
+        self.assertDictEqual(parameters, model._parameters)
+
+        for invalid_param_set in invalid_parameters:
+            with self.assertRaises(ValueError):
+                ColumnNameModel(
+                    parameters=invalid_param_set
+                )
+
+    @mock.patch("sys.stdout", new_callable=StringIO)
+    def test_help(self, mock_stdout):
+        ColumnNameModel.help()
+        self.assertIn("ColumnNameModel", mock_stdout.getvalue())
+        self.assertIn("Parameters", mock_stdout.getvalue())
+
+    @mock.patch("sys.stdout", new_callable=StringIO)
+    def test_predict(self, mock_stdout):
+        parameters = {
+            "true_positive_dict": [
+                {
+					"attribute": "ssn",
+					"label": "ssn"                
+                },
+                {
+                	"attribute": "suffix",
+                	"label": "name"
+                },
+                {
+                	"attribute": "my_home_address",
+                	"label": "address"
+                },
+            ],
+            "false_positive_dict": [
+                {
+                    "attribute": "contract_number",
+                    "label": "ssn",
+                },
+                { 
+                    "attribute": "role",
+                    "label": "name",
+                },
+                {
+                    "attribute": "send_address",
+                    "label": "address",
+                },
+            ]
+        }
+        model = ColumnNameModel(parameters=parameters)
+
+        expected_output = {
+            'ssn': 
+                {
+                    'pred': 'ssn',
+                }
+            }
+
+        with self.assertLogs("DataProfiler.labelers.column_name_model", level="INFO") as logs:
+            model_output = model.predict(data=['ssn', 'role_name', 'wallet_address'])
+
+        self.assertIn("pred", model_output['ssn'])
+        for expected, output in zip(expected_output["ssn"], model_output["ssn"]):
+            self.assertTrue(np.array_equal(expected, output))
+
+        # check verbose logging
+        self.assertTrue(len(logs.output))
+
+        # test show confidences
+        expected_output = {
+            "ssn": 
+                {
+                    "pred": "ssn",
+                    "conf": 100.0
+                }
+            }
+        model_output = model.predict(data=["ssn", "role_name", "wallet_address"], show_confidences=True)
```

Comment:
do we have a test for false as well?

Reply:
yes, the below is a `FALSE` scenario
```
with self.assertLogs(
            "DataProfiler.labelers.column_name_model", level="INFO"
        ) as logs:
            model_output = model.predict(data=["ssn", "role_name", "wallet_address"])
```

Reply:
ahh i missed it, sorry since it wasn't declared i missed it.